.gitignore

```diff
@@ -2,6 +2,7 @@
 /out/
 *.pyc
 gclient_config.py_entries
+Cargo.lock
```

Comment:
Does this have any effect? IIRC this file is in third_party.

Reply:
I believe it got created for me. I think Cargo is run in the root.